contrib/ec2/provision-ec2-cluster.sh

```diff
@@ -98,3 +98,13 @@ echo "Using ELB $ELB_NAME at $ELB_DNS_NAME"
 
 echo_green "Your Deis cluster has been successfully deployed to AWS CloudFormation and is started."
 echo_green "Please continue to follow the instructions in the documentation."
+
+FIRST_INSTANCE=$(aws ec2 describe-instances \
+    --filters Name=tag:aws:cloudformation:stack-name,Values=$STACK_NAME Name=instance-state-name,Values=running \
+    --query 'Reservations[].Instances[].[PublicIpAddress]' \
+    --output text | head -1)
+echo_green "Setting DEISCTL_TUNNEL=$FIRST_INSTANCE"
+export DEISCTL_TUNNEL=$FIRST_INSTANCE
+echo_green "Enabling proxy protocol"
+echo "deisctl config router set proxyProtocol=1"
+deisctl config router set proxyProtocol=1
```

Comment:
It should be enabled per default for AWS.

I think we need this to be part of the provisioning script. BUT, how about checking the return value of `deisctl`?

If it's non-zero then display a 5-10 line warning saying "You have to execute this command otherwise deis-router will experience issues" or something like that. Make it absolutely obvious that it is of high importance.

If it succeeds then great.